src/annotator/config/test/config-test.js

```diff
@@ -0,0 +1,185 @@
+'use strict';
+
+var proxyquire = require('proxyquire');
+
+var fakeSettings = {};
+var fakeSharedSettings = {};
+var configFrom = proxyquire('../config', {
+  './settings': fakeSettings,
+  '../../shared/settings': fakeSharedSettings,
```

Comment:
I'm unit testing `config.js` in isolation from the `setting.js` files. Note that in future I aim to make `config.js` so simple that there'll be next to nothing to test